Representing BO page and related schema (#598)
openownership/data-standard: docs/schema/guidance/repr-beneficial-ownership.rst

.. _representing-bo:

Representing beneficial owners
========================================

.. highlights::

>     **Key requirements:** If a person is a beneficial owner of an entity - whether directly or indirectly - and the person or entity is required to declare this beneficial ownership, there MUST be a Relationship statement connecting the two which represents the beneficial ownership relationship.

Reviewer: yeah that's fair!

    If an entity declares that it has no disclosable beneficial owners, its record MUST be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'noBeneficialOwners'.

Overview
------------------------

> Beneficial owners can exercise their interests *directly* in an entity (expected to be a company) or *indirectly*, via intermediary entities (such as arrangements or other companies). It must be clear in a BODS dataset which people are declared as beneficial owners of which companies (and which entities are intermediaries). And it must be clear what overall interests beneficial owners have (regardless of whether they are direct or indirect).

Reviewer: ```suggestion
Beneficial owners can exercise their interests *directly* in an entity or *indirectly*, via intermediary entities (such as arrangements or companies). It must be clear in a BODS dataset which people are declared as beneficial owners of which companies (and which entities are intermediaries). And it must be clear what overall interests beneficial owners have (regardless of whether they are direct or indirect).
```

.. figure:: ../../_assets/RepresentingChainsBODS-RealWorld.svg

>    :alt: Person 1 indirectly holds a 15 percent shareholding in Company E, via an intermediary: Company A. Person 2 directly holds 32 precent of Company E's shares.

Reviewer: ```suggestion
   :alt: Person 1 indirectly holds a 15 percent shareholding in Company E, via an intermediary: Company A. Person 2 directly holds 32 percent of Company E's shares.
```

* ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
* ``directOrIndirect`` (See :ref:`schema-interest`)
* ``componentRecords`` (See :ref:`schema-relationship-record`)
* ``isComponent`` (See :ref:`schema-entity-record`)

Requirements
------------------------

No beneficial ownership relationships
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> If an entity declares that it has no disclosable beneficial owners, its record MUST be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'noBeneficialOwners'.

Reviewer: ```suggestion
If an entity declares that it has no disclosable beneficial owners, its record MUST be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty` 'noBeneficialOwners'.
```

I think the structure of this has changed and 'reason' is no longer a named field

Reviewer: ahhh my bad I misunderstood that reference

> If an entity is exempt from declaring its beneficial owners its record SHOULD be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'subjectExemptFromDisclosure'.

Reviewer: ```suggestion
If an entity is exempt from declaring its beneficial owners its record SHOULD be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty`` 'subjectExemptFromDisclosure'.
```

as above

A beneficial ownership relationship
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> If a person is a beneficial owner of an entity, entity X, whether directly or indirectly, and one of them is required to declare this beneficial ownership:

Reviewer: ```suggestion
If a person is a beneficial owner of an entity (directly or indirectly) and their relationship is declared there MUST be a *primary* relationship connecting the two which represents the beneficial ownership relationship.
```
I don't think entity X is useful here. The numbering doesn't make sense without point 2 so might be neater to have it all in one sentence. 

should this be Relationship Statement / Relationship Record? Just to clearly distinguish from the irl relationships?

Author: Yes, the 'entity X' thing is unpleasant. I introduced it for clarity because we have to refer back to this entity from the Intermediaries section. If there's a better way to communicate this, let edit.

I've taken your point about the numbering and merged the first two clauses.

Reviewer: Yeah that's fair - I've tried to think of a better way and I can't think of anything significantly better atm

> 1. There MUST be a *primary* relationship connecting the two which represents the beneficial ownership relationship. Specifically, in the ``recordDetails`` for the relationship: 

Reviewer: ```suggestion
In the ``recordDetails`` for the relationship: 
``

>    a. the entity’s record MUST be the ``subject``;

Reviewer: ```suggestion
   * the entity’s ```recordId``` MUST be the ``subject``
```
just to be clear this is what the field should be? removing semi colon as per style guide. Also more consistent with other pages to have bullet points rather than number/letters

Author: Yes - I've changed to 'recordId' - much better.

>    b. the person’s record MUST be the ``interestedParty``;

Reviewer: ```suggestion
   * the person’s ```recordId``` MUST be the ``interestedParty``
```

>    c. ``isComponent`` MUST be false;

Reviewer: ```suggestion
   * ``isComponent`` MUST be false
```

>    d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included if known; and

Reviewer: ```suggestion
   * the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included if known and MUST have ``beneficialOwnershipOrControl`` set to 'true'.
```

Reviewer: think it makes more sense to group these rather than refer to the earlier bullet point from the next one?

Author: Sure.

>    e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'. If an interest is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.

Reviewer: ```suggestion
   * if an interest is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
```

Intermediaries
^^^^^^^^^^^^^^

Where beneficial ownership is known to be exercised indirectly, via known intermediary elements, this SHOULD be represented in addition to the above. In particular:

> 1. The chain of known intermediary elements SHOULD be represented by *secondary* records.

Reviewer: ```suggestion
* the chain of known intermediary elements SHOULD be represented by *secondary* records
```

style guide

> 2. These secondary records SHOULD link the beneficial owner’s record to entity X’s record indirectly.

Reviewer: ```suggestion
* these secondary records SHOULD link the beneficial owner’s record to the root entity's record indirectly
```

> 3. These secondary records SHOULD all have ``isComponent`` set to 'true'.

Reviewer: ```suggestion
* these secondary records SHOULD all have ``isComponent`` set to 'true'
```

> 4. These secondary records SHOULD all have their ``recordId`` values listed in the ``componentRecords`` array of the ``recordDetails`` for the primary relationship.

Reviewer: ```suggestion
* these secondary records SHOULD all have their ``recordId`` values listed in the ``componentRecords`` array of the ``recordDetails`` for the primary relationship
```

> 5. When ``recordDetails`` for the primary relationship are published in a BODS file:

Reviewer: ```suggestion
When ``recordDetails`` for the primary relationship are published in a BODS file:
```

I think this should be a separate point as it is about the publishing not the data itself? Also lists within lists gets a bit complicated 

Also I find the reference to recordDetails a bit confusing here - I think this should be relationship Record / Statement

Author: Yup.

>    a. Statements for all secondary records referenced from ``componentRecords`` MUST also be published in that file;

Reviewer: ```suggestion
   * statements for all secondary records referenced from ``componentRecords`` MUST also be published in that file
```

>    b. Statements for all secondary records must appear before the the first Statement for the primary relationship.

Reviewer: ```suggestion
   * statements for all secondary records must appear before the the first Statement for the primary relationship
```

Reviewer: make sure however the primary relationship is referred to is consistent with the lead in line
